### PR TITLE
next/349/20240315/v1

### PR DESCRIPTION
**Pull request by victorjulien**
Staging:
 - #10623
 - #10636
 - #10639
 - #10642
 - #10645

SV_BRANCH=https://github.com/OISF/suricata-verify/pull/1706

### CODE REVIEW

**Comment by suricata-qa** [comment]
Information:
 

ERROR: QA failed on SURI_TLPW2_autofp_suri_time.
 | field | baseline | test | % |
|--|--|--|--| 
| | SURI_TLPW2_autofp_stats_chk | | |
| .uptime | 138 | 147 | 106.52% | 

Pipeline 19368

File: examples/plugins/c-json-filetype/filetype.c [hunk 1]
```diff
@@ -140,12 +125,12 @@ static void FiletypeDeinit(void *data)
...
  *
- * In the case of non-threaded EVE logging this function is NOT called by
- * Suricata, but instead this plugin chooses to use this method to create a
- * default (single) thread context.
+ * In the case of non-threaded EVE logging this function is called
+ * once with a thread_id of 0.
  */
 static int FiletypeThreadInit(void *ctx, ThreadId thread_id, void **thread_data)
 {
+    SCLogNotice("thread_id=%d", thread_id);
```
Comment (inashivb) [inline examples/plugins/c-json-filetype/filetype.c:130]: are these notice statements supposed to go as is?

Comment (jlucovsky) [reply examples/plugins/c-json-filetype/filetype.c:130]: Yes --- this code is for demonstrative purposes only for an example output plugin.

**Review by jlucovsky — APPROVED** [review]

**Review by jasonish — APPROVED** [review]

**Comment by victorjulien** [comment]
@ct0br0 can this get a rerun? I've tested it locally and I'm not seeing deviations.

**Comment by ct0br0** [comment]
Already have, just waiting on another run to finish

**Comment by victorjulien** [comment]
> Already have, just waiting on another run to finish

Thanks!

**Comment by suricata-qa** [comment]
Information:
 

ERROR: QA failed on SURI_TLPR1_suri_time.
 | field | baseline | test | % |
|--|--|--|--| 
| | SURI_TLPR1_stats_chk | | |
| .uptime | 638 | 679 | 106.43% | 

Pipeline 19374

**Comment by ct0br0** [comment]
Considering that's a different test, I think it's safe to say it's random deviation we've been seeing.

**Comment by victorjulien** [comment]
QA failures. Starting another run.

**Pull request closed without merging**